utils/create-database.js

```diff
@@ -0,0 +1,57 @@
+// require the promise version of mysql2
```

Comment:
Most of the comments in this file are not adding much in the way of value and just state _what_ the line is doing, rather than _why_ it is doing it which would be far more useful in the case of convoluted or difficult work. In this case it isnt, and can be removed without affecting the readability of the code